lxgettext/lxgettext.py

```diff
@@ -0,0 +1,190 @@
+import argparse
+import datetime
+import os
+import re
+
+import polib
+
+COLOUR_GREEN = '\033[92m'
+COLOUR_END = '\033[0m'
+
+KEYWORD = "gettext"
+gettext_re = re.compile("""%s\(['"](.+?)['"]\)""" % KEYWORD)
```

Comment:
true, that is probably why they use a real parsers! ;) we just need to be aware of it